_posts/2020-04-29-newpipe-0.19.3-release.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "NewPipe 0.19.3 released: Android TV, YouTube music, greeting new team members and so much more"
+short: "NewPipe 0.19.3 released"
+date: 2020-04-29 00:00:00 +0100
+categories: [release, pinned]
+author: Poolitzer
+image: newpipe
+excerpt_separator: <!-- more -->
+---
+
+Another day in quarantine, another release of NewPipe. This one comes with a lot of bug fixes, improvements, more features and more NewPipeTeam members. It can not get better then that.
```

Comment:
It's Team NewPipe. Also, not sure if you should mention that here as we aren't new in specifically this releade.